static/include/app/apis/generated/app.md

```diff
@@ -1643,6 +1645,12 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 ```ts {class="line-numbers linkable-line-numbers"}
 const robotPart = await appClient.getRobotPart('<YOUR-ROBOT-PART-ID>');
+// Get the part's address
+const address = robotPart.part.fqdn;
+// Check if machine is live (last access time less than 10 sec ago)
+if (Date.now() - Number(robotPart.part.lastAccess.seconds) * 1000 <= 10000) {
```

Comment:
Frustrating that they only provide seconds and a separate count of nanoseconds on top of that. At least the magic numbers are justified here!